@xen-orchestra/lite/src/stories/web-core/ui-legend-story.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('value').type('number').widget().preset(4).required(),
+      prop('state')
+        .type('string')
+        .preset('default')
+        .enum('default', 'success', 'warning', 'error', 'disabled', 'dark-blue')
+        .widget(),
+      prop('label').type('string').preset('Legend label').widget(),
+      prop('unit').type('string').preset('GB').widget(),
+      prop('tooltip').type('string').widget(),
+    ]"
```

Comment:
- `color` prop should be `required()` (`label` was too).
- `value` prop should not be `required()`.

(I also added a slot configuration. See `UiLegend.vue` proposals below).

```suggestion
    :params="[
      prop('color')
        .type('LegendColor')
        .required()
        .preset('default')
        .enum('default', 'success', 'warning', 'error', 'disabled', 'dark-blue')
        .widget(),
      prop('value').num().widget().preset(4),
      prop('unit').str().preset('GB').widget(),
      prop('tooltip').str().widget(),
      slot(),
      setting('defaultSlotContent').preset('Legend label').widget(text()),
    ]"
```